<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Weekly_Stats_Application</class>
 <widget class="QWidget" name="Weekly_Stats_Application">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Weekly_Stats_Application</string>
  </property>
  <property name="toolTipDuration">
   <number>-5</number>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>551</width>
     <height>122</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="Week_Selection">
    <item row="1" column="1">
     <widget class="QDateEdit" name="week_end">
      <property name="dateTime">
       <datetime>
        <hour>23</hour>
        <minute>59</minute>
        <second>59</second>
        <year>2020</year>
        <month>4</month>
        <day>13</day>
       </datetime>
      </property>
      <property name="maximumDate">
       <date>
        <year>2050</year>
        <month>12</month>
        <day>31</day>
       </date>
      </property>
      <property name="minimumDate">
       <date>
        <year>2018</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>2020</year>
        <month>4</month>
        <day>13</day>
       </date>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="week_end_label">
      <property name="text">
       <string>Week End</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDateEdit" name="week_start">
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2020</year>
        <month>4</month>
        <day>13</day>
       </datetime>
      </property>
      <property name="maximumDateTime">
       <datetime>
        <hour>23</hour>
        <minute>59</minute>
        <second>59</second>
        <year>2050</year>
        <month>12</month>
        <day>31</day>
       </datetime>
      </property>
      <property name="maximumDate">
       <date>
        <year>2050</year>
        <month>12</month>
        <day>31</day>
       </date>
      </property>
      <property name="minimumDate">
       <date>
        <year>2018</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>2020</year>
        <month>4</month>
        <day>13</day>
       </date>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="ok_btn">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="week_start_label">
      <property name="text">
       <string>Week Start</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="ok_btn_label">
      <property name="text">
       <string>Press OK to use selected dates</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>1021</width>
     <height>294</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="Exclusion_Dates">
    <item row="1" column="6">
     <widget class="QLabel" name="sat_label">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sat</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="tues_label">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Tues</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QPushButton" name="fri_3rd_btn">
      <property name="text">
       <string>16:00</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">Exclusion_Dates_btns</string>
      </attribute>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="fri_1st_btn">
      <property name="text">
       <string>00:00</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">Exclusion_Dates_btns</string>
      </attribute>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="mon_label">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Mon</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="fri_label">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Fri</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="wed_label">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Wed</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QPushButton" name="fri_2nd_btn">
      <property name="text">
       <string>08:00</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">Exclusion_Dates_btns</string>
      </attribute>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="wed_1st_btn">
      <property name="text">
       <string>00:00</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">Exclusion_Dates_btns</string>
      </attribute>
     </widget>
    </item>
    <item row="7" column="5" colspan="2">
     <widget class="QPushButton" name="get_data_btn">
      <property name="text">
       <string>Get Data</string>
      </property>
     </widget>
    </item>
    <item row="6" column="5" colspan="2">
     <widget class="QPushButton" name="save_exc_dates_btn">
      <property name="text">
       <string>Save Exclusion Dates</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="thur_label">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Thur</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="thur_1st_btn">
      <property name="text">
       <string>00:00</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">Exclusion_Dates_btns</string>
      </attribute>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="wed_2nd_btn">
      <property name="text">
       <string>08:00</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">Exclusion_Dates_btns</string>
      </attribute>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="tues_2nd_btn">
      <property name="text">
       <string>08:00</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">Exclusion_Dates_btns</string>
      </attribute>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="sun_2nd_btn">
      <property name="text">
       <string>08:00</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">Exclusion_Dates_btns</string>
      </attribute>
     </widget>
    </item>
    <item row="5" column="6">
     <widget class="QPushButton" name="sat_3rd_btn">
      <property name="text">
       <string>16:00</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">Exclusion_Dates_btns</string>
      </attribute>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="sun_1st_btn">
      <property name="text">
       <string>00:00</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">Exclusion_Dates_btns</string>
      </attribute>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="mon_1st_btn">
      <property name="text">
       <string>00:00</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">Exclusion_Dates_btns</string>
      </attribute>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="sun_label">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sun</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="tues_1st_btn">
      <property name="text">
       <string>00:00</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">Exclusion_Dates_btns</string>
      </attribute>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QPushButton" name="sat_1st_btn">
      <property name="text">
       <string>00:00</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">Exclusion_Dates_btns</string>
      </attribute>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="tues_3rd_btn">
      <property name="text">
       <string>16:00</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">Exclusion_Dates_btns</string>
      </attribute>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="mon_3rd_btn">
      <property name="text">
       <string>16:00</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">Exclusion_Dates_btns</string>
      </attribute>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="mon_2nd_btn">
      <property name="text">
       <string>08:00</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">Exclusion_Dates_btns</string>
      </attribute>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="sun_3rd_btn">
      <property name="text">
       <string>16:00</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">Exclusion_Dates_btns</string>
      </attribute>
     </widget>
    </item>
    <item row="4" column="6">
     <widget class="QPushButton" name="sat_2nd_btn">
      <property name="text">
       <string>08:00</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">Exclusion_Dates_btns</string>
      </attribute>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="wed_3rd_btn">
      <property name="text">
       <string>16:00</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">Exclusion_Dates_btns</string>
      </attribute>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="thur_2nd_btn">
      <property name="text">
       <string>08:00</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">Exclusion_Dates_btns</string>
      </attribute>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="thur_3rd_btn">
      <property name="text">
       <string>16:00</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">Exclusion_Dates_btns</string>
      </attribute>
     </widget>
    </item>
    <item row="0" column="0" colspan="7">
     <widget class="QLabel" name="desc_label">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Select shifts to be excluded from the above week, ie Maintenance or Development</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="3" colspan="2">
     <widget class="QLabel" name="save_exc_dates_label">
      <property name="text">
       <string>Press this button to have the excluded dates saved</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3" colspan="2">
     <widget class="QLabel" name="get_data_label">
      <property name="text">
       <string>Press this button to get trip data for week</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QPushButton" name="clear_btn">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="QLabel" name="clear_label">
      <property name="text">
       <string>Clear Exclusion Dates</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>101</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;To use this application first select the date the week started on, then select the date the week ended on. Hit the OK button. Select all shifts that you want excluded from the week. These can be maintenance or development shifts. Select all that apply to that week then hit the Save Exclusion Dates button, that will allow the script to save these dates to be excluded. Once you are satisfied with everything, hit the Get Data button to get the trip data for the week.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="Exclusion_Dates_btns">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
